### ISSUE TITLE
Create Top Level Repo for Killercoda

### ISSUE THREAD

**Issue by agardnerIT** [tektoncd/community#1036]
Hi community, further to these two slack threads:

- https://tektoncd.slack.com/archives/CJ62C1555/p1688432974706169
- https://tektoncd.slack.com/archives/CL3FKC24S/p1689324590827079

I would like to propose that a new top level repo is created called. This repo will house the killercoda tutorial(s) code.

In case the slack threads age out, the TLDR is:

- I created a killercoda (online, in browser) version of the getting started guide
- It is currently here: https://killercoda.com/agardnerit/scenario/tekton but it would be good to donate this to the project and thus move it over to your username.
- Discussions in Slack lead me to believe you used to have Katacoda tutorials so you're aware of the concept.

##  How Much Tech Debt Will This Cause?
This is, of course, another repository and another asset to maintain. But, since this is only a tutorial, it is feasible that its adherence to the latest release and functionality could be allowed to fluctuate.

This repo being slightly out of date will never cause any blockers to tekton releases. That said, new functionality introduced in the "core product" will want to be reflected in this repo ASAP to show the latest and greatest of the current feature set.

In other words, this repo should aim to stay current and closely mirror whatever is in the "getting started" guide.

## Who will maintain this?

As it stands, the tutorial will aim to closely mirror the getting started guide. So it makes sense for whomever maintains the getting started guide, to also maintain this as a single "cohesive unit".

I am willing to assist however and whenever I can in this endeavour. If there needs to be a nominated maintainer for this piece, I'm happy to be it.
